package.json

```diff
@@ -8,11 +8,13 @@
   },
   "scripts": {
     "algolia": "node ./algolia.js",
+    "pre-build": "npm-run-all clean ignore gulp rollup maybe-sync-external",
     "clean": "rimraf dist",
     "cloud-secrets": "node ./cloud-secrets.js",
-    "dev": "npm-run-all clean ignore gulp rollup maybe-sync-external --parallel --race gulp:watch rollup:watch eleventy:watch start",
+    "dev": "npm-run-all pre-build eleventy --parallel start watch",
+    "watch": "npm-run-all --parallel --race gulp:watch rollup:watch eleventy:watch",
```

Comment:
Is [`--race`](https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#:~:text=npm%20run%20bar%27.-,%2Dr%2C%20%2D%2Drace,-%2D%20%2D%20%2D%20%2D%20%2D%20%2D%20%2D%20Set%20the%20flag) needed here? When will any of those `:watch` scripts return a `0` exit status?